src/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { StatelessComponent, useState, useEffect } from 'react';
+import { track } from './utils';
+
+const Footer: StatelessComponent = () => {
+  const [expanded, setExpanded] = useState(false);
+
+  useEffect(() => {
+    track('footer-render');
+  });
```

Comment:
This will fire on every render - should have an array of deps containing track as second argument in hook to only fire once!
```suggestion
  useEffect(() => {
    track('footer-render');
  }, [track]);
```